src/components/Sidebar/Sidebar.test.tsx

```diff
@@ -0,0 +1,75 @@
+import userEvent from "@testing-library/user-event";
+
+import Sidebar from "./Sidebar";
+
+import renderWithTheme from "@/__tests__/__helpers__/renderWithTheme";
+import {
+  mockUnit,
+  mockOptionalityUnit,
+} from "@/components/UnitModal/UnitModal.fixture";
+
+describe("Sidebar component", () => {
+  test("should render the sidebar", () => {
+    const { getByTestId } = renderWithTheme(
+      <Sidebar displayModal={true} onClose={jest.fn()} />,
+    );
+
+    expect(getByTestId("sidebar-modal")).toBeInTheDocument();
+  });
+
+  test("should render the sidebar with children", () => {
+    const { getByTestId } = renderWithTheme(
+      <Sidebar displayModal={true} onClose={jest.fn()}>
+        <div data-testid="sidebar-children" />
```

Comment:
Not sure what this line is doing - does this also get rendered in the component?

Reply:
The sidebar should render the children of it, so this test is to check to see if that div is present